Sming/SmingCore/Network/TcpConnection.h

```diff
@@ -156,10 +156,55 @@ struct SSLKeyCertPair {
 	}
 };
 
-typedef struct {
+/** @brief Manages buffer to store SSL Session ID
+ */
+struct SSLSessionId {
```

Comment:
The TcpConnection.h is starting to have too much code that is not strictly Tcp related but SSL related. Can you put SSLSessionId, SslFingerprintType, SSLFingerprints  and  SSLKeyCertPair into a separate header file?

Reply:
> Can you put SSLSessionId, SslFingerprintType, SSLFingerprints and SSLKeyCertPair into a separate header file?

Sorry, I was not clear enough. I've meant one header file and not header file per structure. Can you put them back to one header file OR create a separate header file in which all new SSL related header files are included?

Reply:
I'll be elaborating on `SSLFingerprints` so thought it best to split them functionally - it's like the 'fingerprint API' header. Yes, can certainly use a common header - is it worth having an `SSL` sub-folder perhaps?